src/platform/plugins/shared/field_formats/common/converters/string.ts

```diff
@@ -114,6 +122,12 @@ export class StringFormat extends FieldFormat {
     if (val === '') {
       return emptyLabel;
     }
+    if (val === '__missing__' || val == null) {
+      return missingLabel;
+    }
+    if (val === '__other__') {
```

Comment:
Where does `__other__` come from?

Reply:
Other comes from Lens `Other` grouping, that is a special string we use to describe all the other terms available.
I was considering aligning these in the same place they works the same